core/src/test/java/org/apache/iceberg/deletes/TestPositionFilter.java

```diff
@@ -110,155 +106,6 @@ public void testPositionMerging() {
         .containsExactly(0L, 3L, 3L, 9L, 16L, 19L, 19L, 22L, 22L, 56L, 63L, 70L, 91L);
   }
 
-  @Test
-  public void testPositionStreamRowFilter() {
```

Comment:
I tend to prefer not removing the tests until the deprecated method is actually removed from the code, but I think in this case it's fine since it's not a typical public API and rather a utility.